README.md

```diff
@@ -46,15 +46,26 @@ Prerequisites:
 - MySQL 8.0.* or (better) Docker
   - For Windows users, install Docker Desktop and then run `docker-compose up` at the root of the project on the command line
 
+### Start locally with only MYSQL running in docker
 
 1. Clone the repository
 2. Import the project as a maven project into your favourite IDE (or run maven on the terminal)
-3. Start the MySQL Database or run the docker-compose file `docker-compose up`
-4. Run `BookProjectApplication.java`
+3. Build the project using `mvn clean install`
+4. Start MYSQL databse using `docker-compose up -d mysql
+5. Start the application using `java -jar target/book-project-0.0.1-SNAPSHOT.jar` 
+6. Log in with the details below:
+    - Username: `user`
+    - Password: `password`
+
+### Start using docker-compose in production mode
+1. Clone the repository
```

Comment:
Could you add a hint that one needs to build the application with the production flag before one can start the application?
`mvn package -Pproduction`

Reply:
sure!